database.sql

```diff
@@ -638,7 +635,7 @@ CREATE TABLE IF NOT EXISTS `manage` (
 	`uid` int(11) NOT NULL DEFAULT 0,
 	`mid` int(11) NOT NULL DEFAULT 0,
 	 PRIMARY KEY(`id`),
-	 INDEX `uid_mid` (`uid`,`mid`)
+	 UNIQUE INDEX `uid_mid` (`uid`,`mid`)
```

Comment:
Have you validated that the combination of `uid` and `mid` can be really unique?